app/src/main/java/com/lightricks/feedexercise/data/FeedRepository.kt

```diff
@@ -1,9 +1,23 @@
 package com.lightricks.feedexercise.data
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Transformations
+import com.lightricks.feedexercise.database.Entity
+import com.lightricks.feedexercise.database.FeedDao
+import com.lightricks.feedexercise.database.FeedDatabase
+
 /**
  * This is our data layer abstraction. Users of this class don't need to know
  * where the data actually comes from (network, database or somewhere else).
  */
-class FeedRepository {
-    //todo: implement
+class FeedRepository(private val feedDao: FeedDao) {
+    val database = FeedDatabase.getExistingDatabase()
```

Comment:
Right now it's not important.
But in step 4 you will need to inject `FeedDatabase` and `FeedApiService` into `FeedRepository` as arguments.